src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -43,20 +62,21 @@ public function __construct( Connection $connection, IsToggleActive $toggleCheck
      */
     public function getAllMyActivatedToggles( $userIdentifier )
```

Comment:
So having sat down with @VolodymyrSergeyev this Pull Request creates a dependency between this Gateway and a UseCase.

It is fine to have UseCases depend on other UseCases, but gateways shouldn't depend on UseCases.